races.py

```diff
@@ -108,14 +108,11 @@ def do_geocode(place, attempts=0):
             place_of_race = p.text.strip('()')
             location = do_geocode(place_of_race)
             if location is None:
-                latitude.append(.0)
-                longitude.append(.0)
+                latitude_column.append(.0)
+                longitude_column.append(.0)
             else:
-                latitude.append(location.latitude)
-                longitude.append(location.longitude)
-            #print(location.latitude)
-            #print(location.longitude)
-            #print(place_of_race)
+                latitude_column.append(location.latitude_column)
```

Comment:
`location.latitude` and `location.longitude` should remain as they were - those are variables defined by Geopy, so you cannot rename them here :-) You can only rename the lists that you are appending into, since those are "your" variables.

Reply:
I am totally lost today :-D